examples/integer-range/profile.json

```diff
@@ -1,15 +1,35 @@
 {
```

Comment:
The indentation amount seems too much for this file.

Reply:
These two files used tabs for indentation whilst the rest used two spaces. Good spot, now changed